pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -275,6 +275,7 @@ self: super: {
   github-rest = dontCheck super.github-rest;  # test suite needs the network
   gitlib-cmdline = dontCheck super.gitlib-cmdline;
   GLFW-b = dontCheck super.GLFW-b;                      # https://github.com/bsl/GLFW-b/issues/50
+  greenclip = markUnbroken (addExtraLibrary super.greenclip pkgs.xlibs.libXScrnSaver);
```

Comment:
Could you instead remove `greenclip` from the file `pkgs/development/haskell-modules/configuration-hackage2nix/broken.yaml`?

Also, do you know why this extra library is needed?  If it should actually be added to the cabal file, could you create an issue upstream?  Or if this is a library that `cabal2nix` doesn't know about, could you send a PR there?

Reply:
this seems like a valid dependency https://github.com/erebe/greenclip/issues/110. I've submitted https://github.com/NixOS/cabal2nix/pull/496 : when this last PR is merged, when can I expect that version of cabal2nix to be usable from nixpkgs ?

Reply:
Immediately after someone runs the "update-cabal2nix-unstable" maintainer script.

Reply:
could I do that in this PR too ? or is it too impactful ?

Reply:
I see no reason against it.